### ISSUE TITLE
No space left on device error on a node farm vlcloudfarm

### ISSUE THREAD

**Issue by delandtj** [threefoldtech/support#5]
there is a node in vlcloudfarm farmer has an issue in space that is full 100% usage, node IP `10.102.28.72` 

```
In [1]: ncl = j.clients.zos.get(instance='node', data = { 'host':'10.102.28.72'})


In [7]: zrobot = ncl.client.container.client(4)

In [8]: job = zrobot.subscribe('zrobot')

In [9]: job.stream()
Traceback (most recent call last):
  File "/.startup.py", line 169, in <module>
    main()
  File "/.startup.py", line 164, in main
    configure_local_client()
  File "/.startup.py", line 28, in configure_local_client
    """)
  File "/opt/code/github/threefoldtech/jumpscale_core/Jumpscale/fs/SystemFSDecorators.py", line 162, in wrapper
    return func(*args, **kwargs)
  File "/opt/code/github/threefoldtech/jumpscale_core/Jumpscale/fs/SystemFS.py", line 1230, in writeFile
    fp.close()
OSError: [Errno 28] No space left on device
Out[9]: 2

In [13]: ncl.client.bash('lsblk').get()
Out[13]: 
STATE: 0 SUCCESS
STDOUT:
NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
sda           8:0    1   3.7G  0 disk 
`-sda1        8:1    1   3.7G  0 part 
nvme0n1     259:0    0 238.5G  0 disk 
`-nvme0n1p1 259:1    0 238.5G  0 part 

STDERR:

DATA:
In [11]: ncl.client.bash('df -kh').get()
Out[11]: 
STATE: 1001 ERROR
STDOUT:
Filesystem                Size      Used Available Use% Mounted on
tmpfs                   640.0M    640.0M         0 100% /
devtmpfs                  7.3G         0      7.3G   0% /dev
cgroup_root               7.3G         0      7.3G   0% /sys/fs/cgroup
overlay                 640.0M    640.0M         0 100% /mnt/containers/4
tmpfs                   640.0M    640.0M         0 100% /mnt/containers/4/opt/code/local/stdorg/config
tmpfs                   640.0M    640.0M         0 100% /mnt/containers/4/opt/var/data/zrobot/zrobot_data
tmpfs                   640.0M    640.0M         0 100% /mnt/containers/4/root/jumpscale/cfg
tmpfs                   640.0M    640.0M         0 100% /mnt/containers/4/root/.ssh
tmpfs                   640.0M    640.0M         0 100% /mnt/containers/4/tmp/redis.sock
tmpfs                   640.0M    640.0M         0 100% /mnt/containers/4/coreX

STDERR:
df: /var/cache/containers/1/a1310e764d51932b8b25ffbf196380cc/ro: Transport endpoint is not connected
df: /var/cache/containers/2/a1310e764d51932b8b25ffbf196380cc/ro: Transport endpoint is not connected
df: /var/cache/containers/3/a1310e764d51932b8b25ffbf196380cc/ro: Transport endpoint is not connected

DATA:
```

**Comment by zaibon** [comment]
Ok so the error is that ssd disk is not used for filesystem cache. Will need to investigate to find out why.

**Comment by Pishoy** [comment]
fixed now by booting zos version  `1.5.0-rc1 @Revision:9fe77c34ba3866552ac1fd339b809133f6ea025e`

```
In [2]: ncl.client.bash('df -kh').get()
Out[2]: 
STATE: 0 SUCCESS
STDOUT:
Filesystem                Size      Used Available Use% Mounted on
tmpfs                   512.0M    324.9M    187.1M  63% /
devtmpfs                  7.3G         0      7.3G   0% /dev
cgroup_root               7.3G         0      7.3G   0% /sys/fs/cgroup
/dev/nvme0n1p1          238.5G    176.7M    237.3G   0% /mnt/storagepools/sp_zos-cache
/dev/nvme0n1p1          238.5G    176.7M    237.3G   0% /var/cache
/dev/nvme0n1p1          238.5G    176.7M    237.3G   0% /var/log
```
